### PR TITLE
fix js.Value argument in DrawImage*

### PR DESCRIPTION
**Pull request by slon**
Fix issue similar to one fixed in https://github.com/dominikh/go-js-dom/commit/26252dedbe70cb87ce3bef8c921f8609982a74f6

### CODE REVIEW

File: v2/dom.go [hunk 1]
```diff
@@ -25,15 +25,13 @@
 // MDN Web Docs by Mozilla Contributors (https://developer.mozilla.org/en-US/docs/Web/API),
 // licensed under CC-BY-SA 2.5 (https://creativecommons.org/licenses/by-sa/2.5/).
 //
-//
-// Getting started
+// # Getting started
```
Comment (slon) [inline v2/dom.go:28]: These changes were done automatically by gofmt

File: v2/dom.go [hunk 2]
```diff
@@ -2395,15 +2391,15 @@ func (ctx *CanvasRenderingContext2D) ResetTransform() {
 // Drawing images
```
Comment (dmitshur) [inline v2/dom.go:2391]: I can't leave the comment on the right line on GitHub, but it seems `CreatePattern` needs the same fix too.

File: v2/dom.go (hunk 1 above)
Comment (dmitshur) [reply v2/dom.go:28]: Applied them separately, in commit 3bbdb75377065c9b06d14b9de35fc5184701e5a1.

**Review by dmitshur — APPROVED** [review]
Thanks.

**Pull request closed without merging**